<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#fundo {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, 
                                stop:0 #E3F2FD, stop:1 #AFCDE7);
    border: 1px solid #6B8EBE;
    border-radius: 4px;
}

#fundo_2 {
	background-color: #f4f4f4;
    border: 1px solid #ccc;
    padding: 5px;
}

#btnCancelar {
    background-color: #ea3323;
    border: none;
    border-radius: 15px;
    color: #ffffff;
    padding: 10px 20px;
}

#btnCancelar:hover {
    background-color: #d62f20;
}

#btnOk {
    background-color: #01498a;
    border: none;
    border-radius: 15px;
    color: #ffffff;
    padding: 10px 20px;
}

#btnOk:hover {
    background-color: #f6921e;
}

#btnAbrirConflitos {
    background-color: #01498a;
    border: none;
    border-radius: 10px;
    color: #ffffff;
    padding: 8px 8px;
}

#btnAbrirConflitos:hover {
    background-color: #f6921e;
}

#btnFechar {
    background-color: #ea3323;
    border-radius: 10px;
    color: #ffffff;
    padding: 8px 16px;
}

#btnFechar:hover {
    background-color: #d62f20;
}

#aviso {
    padding: 10px 20px;
    background-color: #ffffff;
    border-radius: 10px;
    color: #333333;
    line-height: 1.5;
    border: 1px solid #ccc;
    margin-bottom: 15px;
}

#aviso strong {
    color: #ea3323;
}

QLineEdit, QPlainTextEdit {
    background-color: #ffffff;
    border: 1px solid #ccc;
    border-radius: 10px;
    padding: 10px;
}

QLineEdit:focus, QPlainTextEdit:focus {
    border-color: #01498a;
}

QPushButton {
    border-radius: 15px;
    color: #ffffff;
    padding: 10px 20px;
}

QPushButton:hover {
    background-color: #f6921e;
}

QPushButton#btnCancelar {
    background-color: #ea3323;
}

QPushButton#btnOk {
    background-color: #01498a;
}

QPushButton#btnAbrirConflitos {
    background-color: #01498a;
}

QPushButton#btnFechar {
    background-color: #ea3323;
}

QPushButton#btnCancelar:hover {
    background-color: #d62f20;
}

QPushButton#btnOk:hover {
    background-color: #f6921e;
}

QPushButton#btnAbrirConflitos:hover {
    background-color: #f6921e;
}

QPushButton#btnFechar:hover {
    background-color: #d62f20;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="fundo">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item alignment="Qt::AlignRight">
       <widget class="QPushButton" name="btnFechar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>45</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>27</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icones/iconClosePreto.png</normaloff>icones/iconClosePreto.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="fundo_2">
        <property name="minimumSize">
         <size>
          <width>662</width>
          <height>253</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>662</width>
          <height>253</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QFrame" name="frame_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>159</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item alignment="Qt::AlignVCenter">
             <widget class="QLabel" name="label_4">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>65</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>65</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>icones/iconAjuda.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_7">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QLabel" name="titulo">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="aviso">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>56</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="btnAbrirConflitos">
              <property name="minimumSize">
               <size>
                <width>105</width>
                <height>36</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>105</width>
                <height>36</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Ver Conflitos</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="containerEquipamentos">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QCheckBox" name="checkBoxEquipamentos">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Levar Equipamentos</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnOk">
              <property name="minimumSize">
               <size>
                <width>105</width>
                <height>36</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>105</width>
                <height>36</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Sim</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnCancelar">
              <property name="minimumSize">
               <size>
                <width>105</width>
                <height>36</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Cancelar</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
